packages/iTwinUI-react/src/core/Table/TableCell.tsx

```diff
@@ -47,10 +47,13 @@ export const TableCell = <T extends Record<string, unknown>>(
   };
 
   const cellElementProps = cell.getCellProps({
-    className: cx('iui-cell', cell.column.cellClassName),
+    className: cx('iui-cell', cell.column.cellClassName, {
+      'iui-cell-sticky': cell.column.sticky,
```

Comment:
does it work well? Maybe add `!!` so it is actually boolean

Reply:
It worked perfectly but still added